gemrb/core/GUI/WindowManager.cpp

```diff
@@ -43,7 +42,8 @@ Holder<Sprite2D> WindowManager::CursorMouseDown;
 void WindowManager::SetTooltipDelay(int delay)
 {
 	// Max setting disables tooltips in originals, we just set it to an extremely high value for simplicity.
-	ToolTipDelay = (delay == 10 * Tooltip::DELAY_FACTOR) ? 1e7 : delay;
```

Comment:
No no, I meant the `1e7` double, of course named constants are fine.

Reply:
okay